### DISCUSSION TITLE
When using https as a URL in a callback the callback does not appear to be called

### DISCUSSION THREAD

**Discussion by simonhitachi** [mockoon/mockoon#1272] [General]
I have set up some mocking with callbacks on some of the routes as the callbacks reference the same mock server they initially worked when using http. When I changed this to https the callback did not appear to be called (there was no reference to this in the Mockoon UI logs). I even tried pointing this to an external API service using HTTPS and had the same behaviour. One point of note, the endpoint is self-signed.

Is it actually possibly to use a HTTPS callback? When creating a new callback the example URL shown is https, but currently can't get anything other than http requests to get triggered via  a callback.

Running Mockoon v6.1.0 on Windows 10.

**Comment by 255kb** [comment]
Calling HTTPS endpoints through the callbacks feature works, however, it dosn't support self-signed certificates as it is always checking their validity. 
I suggest that you open a new issue to track this. As Mockoon is a dev/testing tool, it makes sense to allow self-signed certs.
